setup.py

```diff
@@ -1,19 +1,33 @@
-"""
-This is a setup.py script generated by py2applet
+from setuptools import setup, find_packages
 
-Usage:
-    python setup.py py2app
-"""
-
-from setuptools import setup
-
-APP = ['dashboard.py']
-DATA_FILES = []
-OPTIONS = {}
+NAME = 'OpenEIT_Dashboard'
+VERSION = '0.1.0'
+AUTHOR = 'Jean Rintoul'
+LICENSE = 'LICENSE'
+INSTALL_REQUIRES = [
+                    'cycler==0.10.0',
+                    'decorator==4.2.1',
+                    'imageio==2.2.0',
+                    'matplotlib==2.1.1',
+                    'networkx==2.0',
+                    'numpy==1.14.0',
+                    'Pillow==5.0.0',
+                    'pyparsing==2.2.0',
+                    'pyserial==3.4',
+                    'python-dateutil==2.6.1',
+                    'pytz==2017.3',
+                    'PyWavelets==0.5.2',
+                    'scikit-image==0.13.1',
+                    'scipy==1.0.0',
+                    'six==1.11.0',
+                    
+                ]
 
 setup(
-    app=APP,
-    data_files=DATA_FILES,
-    options={'py2app': OPTIONS},
-    setup_requires=['py2app'],
+    name=NAME,
+    version=VERSION,
+    author=AUTHOR,
+    license=LICENSE,
+    packages = find_packages(),
```

Comment:
Have you tested that find_packages only finds the openeit package (there is a lot of cruft in the repo).

Reply:
yeah, there is a whole folder of semi-useful cruft that could later be deleted(it's useful for development of new algorithms however but not formalized in any way).

Reply:
Yes, I agree that it is useful to keep it for the moment, since it contains extra testing data and useful "cookbook" like parts for further work.

Reply:
This type of problem can be addressed with passing kwargs to `find_packages(where=, exclude=)`.